src/ui/Button/index.ts

```diff
@@ -6,3 +6,5 @@ export { default as TextButton } from './TextButton/TextButton';
 export { default as TextButtonLink } from './TextButton/TextButtonLink';
 export { default as ShuffleButton } from './ShuffleButton/ShuffleButton';
 export { default as PlayButton } from './PlayButton/PlayButton';
+export { default as MenuButton } from './MenuButton/MenuButton';
+export { default as TrackShareButton } from './MenuButton/TrackShareButton/TrackShareButton';
```

Comment:
```suggestion
export { default as TrackShareButton } from './MenuButton/TrackShareButton/TrackShareButton';
```

`MenuButton` это же по факту промежуточный компонент, поэтому экспортировать наверх его не нужно